UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -10,8 +10,13 @@
 [RequireComponent(typeof(HealthStateMonitor))]
 public abstract class LivingHealthBehaviour : NetworkBehaviour
 {
-	public float maxHealth = 100;
 
+	/// <summary>
+	/// Server side, each mob has a different one and never it never changes
+	/// </summary>
+	public int mobID;
```

Comment:
well, if it never changes then why make it publically editable?
```suggestion
	public int MobID => mobID;
	private int mobID;
```